code/go/0chain.net/smartcontract/dbs/event/allocation.go

```diff
@@ -43,6 +43,7 @@ type Allocation struct {
 	SuccessfulChallenges     int64         `json:"successful_challenges"`
 	FailedChallenges         int64         `json:"failed_challenges"`
 	LatestClosedChallengeTxn string        `json:"latest_closed_challenge_txn"`
+	Users []Users `gorm:"foreignKey:UserID;references:Owner"`
```

Comment:
The array of users look out of place to me. Is there something special about it? Each allocation is has (and belongs to) only one owner (User). Am I missing something here?

Reply:
I have change it to `Users Users` instead of `Users []Users`